build.py

```diff
@@ -1279,6 +1279,23 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
     rm -rf /var/lib/apt/lists/*
 """
 
+    # Build conda environment for vllm backend
+    if "vllm" in backends:
+        df += install_miniconda(argmap["CONDA_VERSION"], target_machine)
+        vllm_conda_env_bash = "https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/tools/environment.yml"
+
+        # The last copy is needed so that user could run vllm models with unpacked
+        # conda env
+        df += """
+# conda environment is required for the vllm backend
+RUN conda install -y mamba -c conda-forge  && \
+    wget "{vllm_conda_env_bash}" && \
+    mamba env create -f environment.yml && \
```

Comment:
I am looking into another ways than providing `environment.yml` with hard coded package versions.

Reply:
Should we start with environment.yml and then change if we find something better?

Reply:
Ok. Let's do this.